R/build-home-community.R

```diff
@@ -0,0 +1,32 @@
+has_contributing <- function(path = ".") {
+  file_exists(path(path, 'CONTRIBUTING.html'))
+}
+
+has_coc <- function(path = ".") {
+  file_exists(path(path, 'CODE_OF_CONDUCT.html'))
+}
+
+data_home_sidebar_community <- function(pkg){
+
+  links <- NULL
+
+  if (has_contributing(pkg$dst_path)) {
+    links <- c(links,
```

Comment:
Can you please use standard tidyverse style here? https://style.tidyverse.org/syntax.html#long-lines